notebooks/112-pytorch-post-training-quantization-nncf/112-pytorch-post-training-quantization-nncf.ipynb

```diff
@@ -107,11 +107,7 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    model_ir = mo.convert_model(model, input_shape=[-1, 3, *IMAGE_SIZE])
</pre><p>for consistency between original and quantized models, can we both models export to onnx?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1058/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>